twitter_stream/models.py

```diff
@@ -299,15 +300,18 @@ def count_approx(cls):
         Get the approximate number of tweets.
         Executes quickly, even on large InnoDB tables.
         """
-        query = "SHOW TABLE STATUS WHERE Name = %s"
-        cursor = connection.cursor()
-        cursor.execute(query, [cls._meta.db_table])
+        if django_settings.DATABASES['default']['ENGINE'].endswith('mysql'):
+            query = "SHOW TABLE STATUS WHERE Name = %s"
+            cursor = connection.cursor()
+            cursor.execute(query, [cls._meta.db_table])
 
-        desc = cursor.description
-        row = cursor.fetchone()
-        row = dict(zip([col[0].lower() for col in desc], row))
+            desc = cursor.description
+            row = cursor.fetchone()
+            row = dict(zip([col[0].lower() for col in desc], row))
 
-        return int(row['rows'])
+            return int(row['rows'])
+        else:
+            return len(cls.objects.all())
```

Comment:
I merged it to the `postgresql` branch. You can pull that, fix this, and submit another PR. Thank you!